Gemfile

```diff
@@ -28,61 +26,19 @@ if RUBY_VERSION >= '2.4.0'
   gem 'json', '>= 2.0.2'
 end
 
-if RUBY_VERSION < '1.9'
-  gem 'ffi', '< 1.9.19' # ffi dropped Ruby 1.8 support in 1.9.19
-else
-  gem 'ffi', '~> 1.9.25'
-end
-
-if RUBY_VERSION >= '2.0.0'
-  gem 'rake', '>= 10.0.0'
-elsif RUBY_VERSION >= '1.9.3'
-  gem 'rake', '< 12.0.0' # rake 12 requires Ruby 2.0.0 or later
-else
-  gem 'rake', '< 11.0.0' # rake 11 requires Ruby 1.9.3 or later
-end
+gem 'ffi', '~> 1.9.25'
```

Comment:
Whats the latest version?

Reply:
Did we figure this out?

Reply:
I'm going to relax this pin to `~> 1.9` which will put us on the latest 1.x series (1.11 is the latest version anyway)